storage/mysql/log_storage.go

```diff
@@ -95,19 +95,19 @@ func NewLogStorage(id trillian.LogID, dbURL string) (storage.LogStorage, error)
 }
 
 func (m *mySQLLogStorage) getLeavesByIndexStmt(num int) (*sql.Stmt, error) {
-	return m.getStmt(selectLeavesByIndexSql, num)
+	return m.getStmt(selectLeavesByIndexSql, num, "?", "?")
 }
 
 func (m *mySQLLogStorage) getLeavesByHashStmt(num int, orderBySequence bool) (*sql.Stmt, error) {
 	if orderBySequence {
-		return m.getStmt(selectLeavesByHashOrderedBySequenceSQL, num)
+		return m.getStmt(selectLeavesByHashOrderedBySequenceSQL, num, "?", "?")
```

Comment:
Might be better to make the code that handles expanding placeholders more general. I think this will do for the moment though.